docs/docs/production-app.md

```diff
@@ -95,15 +95,15 @@ This bundle is produced from [production-app.js](https://github.com/gatsbyjs/gat
 
 This contains the small [webpack-runtime](https://webpack.js.org/concepts/manifest/#runtime) as a separate bundle (configured in `optimization` section). In practice, the app and webpack-runtime are always needed together.
 
-##### framework-[contenthash].js
+#### framework-[contenthash].js
```

Comment:
It doesn't make sense to me that these sections are "under" the `webpack-runtime-[contenthash].js` section, but also this page is outdated and I don't think we care too much? What do you think, @sidharthachatterjee

Reply:
I agree, this should be an h3 to match the others. Same with the following two headings, right?
 
```suggestion
### framework-[contenthash].js
```